googlechromelabs/ise-web-security-bundle/1.0/config/packages/ise_web_security.yaml

```diff
@@ -0,0 +1,13 @@
+ise_web_security:
+        defaults:
+                coop:
+                        active: true
+                        policy: 'same-origin'
+                coep: 
+                        active: true
+                        policy: 'require-corp'
+                fetch_metadata:
+                        active: true
+                        allowed_endpoints: []
+                trusted_types:
+                        enabled: false
```

Comment:
It looks to me like your indentation is always 8 spaces, but if I understand the [requirements](https://github.com/symfony/recipes/blob/master/README.rst#validation) correctly it should only be 4.

> YAML files must use 4 space indentations;

>

Reply:
Good catch! Had vs-code's tab-to-spaces setup wrong on accident. Thanks!